src/Signin.tsx

```diff
@@ -117,9 +113,7 @@ export class Signin extends PureComponent<Props, State> {
        * Quartz is integrated into that environment
        */
       if (CLOUD && isFlagEnabled('authSessionCookieOn')) {
-        const url = new URL(
-          `${window.location.origin}${CLOUD_LOGIN_PATHNAME}?redirectTo=${window.location.href}`
-        )
+        const url = new URL(`${CLOUD_URL}?redirectTo=${window.location.href}`)
```

Comment:
🤷🏽 this should work w/r/t to directing the user the Quartz login page, but I don't think the `redirectTo` stuff will be relevant. Since we're dealing with authentication and signing in, I would feature flag this behind a public feature flag so that we ensure that there's some sort of stability in place prior to merging in something that may unintentionally break auth